systems/robotInterfaces/@Biped/Biped.m

```diff
@@ -221,6 +221,20 @@
     function bool = isRightSupport(obj,rigid_body_support_state)
       bool = ~any(rigid_body_support_state.bodies==obj.robot.foot_body_id.left) && any(rigid_body_support_state.bodies==obj.robot.foot_body_id.right);
     end
+
+    function fc = getFootContacts(obj, q)
```

Comment:
Yup, done. Actually, this method is only ever called by the DRC LCMBroadcastBlock, so we could consider not including it in Biped at all, but this still seemed like a reasonable place to put it.